custom/icds/templates/icds/manage_hosted_ccz.html

```diff
@@ -44,13 +44,13 @@ <h1>
           <th>
             {% trans 'Version' %}
           </th>
-          <th>
-            {% trans 'Tag' %}
+          <th class="col-sm-2">
```

Comment:
just curious what does that change about the column widths when displayed?

Reply:
Without this, "Functional Version" only gets a width of 1 and gets displayed on two lines.

Reply:
cool, and does that also effect other columns? i see you removed it from where i added it earlier for the same reason, Notes can be longer, if added.

Reply:
Yeah, it shrinks the notes column. I was tempted to remove that column altogether, since no one has used it in the year this feature has existed, but thought that might ruffle people's feathers.

Reply:
It was specially requested!
I will drop a note to others to check if we can remove it.